pkg/test/runner/runner.go

```diff
@@ -206,7 +206,17 @@ func (r *Runner) runFnRender() error {
 	// run function
 	kptArgs := []string{"fn", "render", pkgPath}
 	for i := 0; i < r.testCase.Config.RunCount(); i++ {
-		output, fnErr := runCommand("", r.kptBin, kptArgs)
+		output, stderr, fnErr := runCommand("", r.kptBin, kptArgs)
+		if fnErr != nil {
```

Comment:
I will suggest to not compare the results here but delegate it to `r.compareResult`.

Reply:
done, thanks!